### ISSUE TITLE
-temp-dir option does not work

### ISSUE THREAD

**Issue by talha131** [transitive-bullshit/ffmpeg-concat#22]
Even if I pass `--temp-dir /Volumes/Backup/` to the command, like so,

```bash
ffmpeg-concat -d 5000 --temp-dir /Volumes/Backup/ -o ~/Downloads/26b-45min.mp4  26B.mp4 26B.mp4 26B.mp4 26B.mp4
```
`ffmpeg-contact` still creates raw files inside macOS tmp folder

```
{ cmd:
   'ffmpeg -i 26B.mp4 -y -pix_fmt rgba -start_number 0 /private/var/folders/2g/td4j8pj91wb1dn3fszcxmq3h0000gn/T/5f61828e8cfca17b15f2ba8235e4d29b/scene-0-%012d.raw' }
{ cmd:
   'ffmpeg -i 26B.mp4 -y -pix_fmt rgba -start_number 0 /private/var/folders/2g/td4j8pj91wb1dn3fszcxmq3h0000gn/T/5f61828e8cfca17b15f2ba8235e4d29b/scene-1-%012d.raw' }
{ cmd:
   'ffmpeg -i 26B.mp4 -y -pix_fmt rgba -start_number 0 /private/var/folders/2g/td4j8pj91wb1dn3fszcxmq3h0000gn/T/5f61828e8cfca17b15f2ba8235e4d29b/scene-2-%012d.raw' }
```

# Details

```yaml
ffmpeg-concat: 1.0.9
macOS: 10.13.5 (17F77)
```

--

Thank you for creating this awesome product. Other workarounds for creating transition using ffmpeg are so convoluted. Love your solution.

**Comment by transitive-bullshit** [comment]
Interesting; I can reproduce the issue on my end with `v1.0.9`. It seems this version was published without the `--temp-dir` flag.

I just published `v1.0.10` and can verify that it works as intended.

Thanks!!

**Comment by talha131** [comment]
Yeah, issue is fixed in v1.0.10. Thank you for the prompt response.
